packages/analytics/analytics-chart/src/components/AnalyticsChart.vue

```diff
@@ -1,23 +1,22 @@
 <template>
   <div
     class="analytics-chart-shell"
-    :class="{ 'is-hovering': isHovering }"
-    @mouseenter="handleMouseEnter"
-    @mouseleave="handleMouseLeave"
   >
     <div
+      v-if="hasValidChartData && resultSetTruncated && maxEntitiesShown || (!hasKebabMenuAccess && (allowCsvExport || props.chartTitle))"
       class="chart-header"
     >
       <div
-        v-if="chartTitle"
```

Comment:
This `v-if` still needs to be there to avoid rendering an extra div.

Reply:
done